build.assets/makefiles/master/k8s-master/kube-controller-manager.service

```diff
@@ -14,12 +14,15 @@ ExecStart=/usr/bin/kube-controller-manager \
         --master=https://${KUBE_APISERVER}:6443 \
         --logtostderr=true \
         --kubeconfig=/etc/kubernetes/scheduler.kubeconfig \
+        --authentication-kubeconfig=/etc/kubernetes/scheduler.kubeconfig \
+        --authorization-kubeconfig=/etc/kubernetes/scheduler.kubeconfig \
         --profiling=false \
         --terminated-pod-gc-threshold=500 \
-        --port=0 \
```

Comment:
IIRC we set the --port=0 to disable the insecure port. Why does this need to be re-enabled?